backend/server/issue.go

```diff
@@ -40,6 +42,16 @@ func (s *Server) registerIssueRoutes(g *echo.Group) {
 			return echo.NewHTTPError(http.StatusInternalServerError, "Failed to create issue").SetInternal(err)
 		}
 
+		if s.MetricReporter != nil {
+			s.MetricReporter.Report(&metric.Metric{
```

Comment:
The metric is about executing the first schema update task successfully instead of creating the issue. The user needs to execute the task to better understand our value.

Reply:
Add another metric for issue status update